OneSignalSDK/onesignal/core/src/main/java/com/onesignal/session/internal/outcomes/impl/OutcomeEventParams.kt

```diff
@@ -7,7 +7,8 @@ internal class OutcomeEventParams constructor(
     val outcomeId: String,
     val outcomeSource: OutcomeSource?, // This field is optional
     var weight: Float, // This field is optional.
-    var timestamp: Long = 0,
+    var sessionTime: Long, // This field is optional
+    var timestamp: Long,
```

Comment:
`timestamp` previously set its default value to 0. Was removing this default intentional?

Reply:
Yes it was intentional, it was briefly mentioned in a commit but here is more information about why I changed it. 

The SDK actually always passed in a value for `timestamp` to the `OutcomeEventParams` constructor, so this default was not needed. In addition, it was not to confuse with the new `sessionTime` parameter. After adding `sessionTime` to the constructor, the app and unit tests built because the argument we used to pass for `timestamp` is now interpreted as `sessionTime` and `sessionTime` started defaulting to `0`.

But I think it is a good note what this parameter means, that it starts out `0` until it fails and becomes cached.